pkg/api/kyverno/v1/types.go

```diff
@@ -239,9 +239,12 @@ type ResourceDescription struct {
 // Mutation describes the way how Mutating Webhook will react on resource creation
 type Mutation struct {
 	// Specifies overlay patterns
-	Overlay interface{} `json:"overlay,omitempty" yaml:"overlay,omitempty"`
+	Overlay interface{} `json:"overlay,omitempty"`
 	// Specifies JSON Patch
 	Patches []Patch `json:"patches,omitempty" yaml:"patches,omitempty"`
```

Comment:
would be good to add comments that this is preserved for backwards compatibility and will be removed in Kyverno 1.5+.